src/librustdoc/clean/types.rs

```diff
@@ -1933,7 +1933,7 @@ crate struct Enum {
 #[derive(Clone, Debug)]
 crate enum Variant {
     CLike,
-    Tuple(Vec<Type>),
+    Tuple(Vec<Item>),
```

Comment:
Why is this a full Item? You only ever use the `ItemKind`.

Reply:
Oups, it's completely unneeded, you're absolutely right.